qir-alliance/pyqir
# Add licensing checks and notice file

- CI pipeline to verify license compliance
- build task to check license compliance
- build task to generate notice file
- config files and templates for notice file
- generated notice file
